share/spice/sales_tax/zipcodes.yml

```diff
@@ -0,0 +1,52 @@
+Alabama: 36101
+Alaska: 99801
+Arizona: 85001
+Arkansas: 72201
+California: 94203
+Colorado: 80123
+Connecticut: 06101
+Delaware: 19901
+District of Columbia: 20001
+Washington D.C: 20001
+Florida: 32301
+Georgia: 30301
+Hawaii: 96813
+Idaho: 83701
+Illinois: 62701
+Indiana: 46201
+Iowa: 50301
+Kansas: 66101
+Kentucky: 40601
+Louisiana: 70801
+Maine: 04330
+Maryland: 21401
+Massachusetts: 02108
+Michigan: 48901
+Minnesota: 55101
+Mississippi: 39201
+Missouri: 65101
+Montana: 59601
+Nebraska: 68501
+Nevada: 89701
+New Hampshire: 94518
+New Jersey: 08601
+New Mexico: 87501
+New York: 12201
+North Carolina: 27601
+North Dakota: 58501
+Ohio: 43085
+Oklahoma: 73101
+Oregon: 97301
+Pennsylvania: 17101
+Rhode Island: 02901
+South Carolina: 29201
+South Dakota: 57501
+Tennessee: 37201
+Texas: 73301
+Utah: 84101
+Vermont: 05601
+Virginia: 23218
+Washington: 98501
+West Virginia: 29401
+Wisconsin: 53705
+Wyoming: 82001
```

Comment:
This is a workaround because the API does not just take a state name. It has an endpoint that accepts a postal code, so that's what I'm using 
http://taxratesapi.avalara.com/docs